data/kamvas-pro-13.tablet

```diff
@@ -0,0 +1,51 @@
+# Huion
+# Kamvas Pro 13
+# GT-133
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#          *-----------------------*
+#          |                       |
+#    A     |                       |
+#    B     |                       |
+#          |                       |
+#          |        DISPLAY        |
+#          |                       |
+#    C     |                       |
+#    D     |                       |
+#    E     |                       |
+#          *-----------------------*
+#
+# Touch Strip Map:
+# NOTE: not enabled because i couldnt get it to work
+# evtest reports 0x108 (BTN_8), 0x109 (BTN_9), 0x130 (BTN_SOUTH)
```

Comment:
touch strips only work as such if they send proper events, in this case they just seem to be button events - we don't have the code for that anywhere (neither the xf86-input-wacom driver nor libinput). So that's fine, fsvo "fine".